modules/core/package.json

```diff
@@ -68,6 +68,7 @@
     "ecurve": "^1.0.6",
     "eol": "^0.5.0",
     "eosjs": "^16.0.8",
+    "eosjs-latest": "npm:eosjs@^21.0.2",
```

Comment:
Why are both 21.02 and 16.x are used? I will be very skeptical of releasing this code in production which uses 2 different versions.

Reply:
eosjs v21 [doesn't give us a transaction_id](https://github.com/BitGo/BitGoJS/pull/1491/files?authenticity_token=nwuvBW71B%2FnfL%2FHVWhhS4hVYcAjuaS5otOZS6PpB9AxM0XV2vX%2Bl2ILMffSVQJ9ZrymGM5mMpneQ7FXNzMNvjg%3D%3D&file-filters%5B%5D=.ts#diff-6efbc825de1f8d6f3fae2bfff41cf42cc40beae8d0af3309e52e11582ab83d9fR579-R583) when we create a transaction since it's not supported anymore. I couldn't find any way to create transaction_id in v21. 

v16 is only being used in 2 places, I was able to refactor all other usages to v21.

Reply:
@shiksb I ran into this problem for tokens and saw that transactionId is a derived value:
From EOS [docs](https://developers.eos.io/welcome/v2.0/protocol/transactions_protocol/#21-transaction-id):
> Consequently, a transaction ID consists of a cryptographic hash of the basic fields included in a transaction instance. Therefore, the transaction ID is solely determined by the list of actions encapsulated within the transaction, the transaction header, and any embedded transaction extensions, which are optional.

We can see how transaction_id is generated by looking at v16's source code for the `transaction` func: https://github.com/EOSIO/eosjs/blob/v16.0.8/src/write-api.js#L496

I verified with the following in `deserializeTransaction`:
```
      if (headers) {
        const rebuiltTransaction = yield api.transact({ ...tx, ...headers }, { sign: false, broadcast: false });
        const transactionId = crypto
          .createHash('sha256')
          .update((rebuiltTransaction as any).serializedTransaction)
          .digest()
          .toString('hex');
        tx.transaction_id = transactionId;
      }
```